text-ui-test/EXPECTED.TXT

```diff
@@ -1,6 +1,7 @@
 Welcome to LongAh!
+Error reading saved PIN and authentication enabled state.
 Thanks for choosing LongAh! Never worry about owing money during the Year of the Dragon!
 Create your 6-digit PIN:
 
 PIN saved successfully!
-Enter your PIN: Enter command:
\ No newline at end of file
+Enter command:
```

Comment:
have some problems with text ui testing in ur ver but I wld hv fixed it so dw abt it